velox/dwio/parquet/tests/ParquetWriterTestBase.h

```diff
@@ -35,17 +36,23 @@ class ParquetWriterTestBase : public testing::Test {
     rootPool_ = memory::defaultMemoryManager().addRootPool("ParquetWriterTest");
     leafPool_ = rootPool_->addLeafChild("SinkTest");
     tempPath_ = exec::test::TempDirectoryPath::create();
+    rowType_ =
+        ROW({"c0", "c1", "c2", "c3", "c4", "c5"},
+            {BIGINT(), INTEGER(), SMALLINT(), REAL(), DOUBLE(), VARCHAR()});
   }
 
   std::vector<RowVectorPtr> createBatches(
-      const std::shared_ptr<const Type>& rowType,
       uint64_t batchNum,
-      uint64_t capacity) {
+      uint64_t vectorSize) {
+    VectorFuzzer::Options opts;
+    opts.constantRatio = 0.0;
+    opts.dictionaryRatio = 0.0;
```

Comment:
I feel there are so many options in the Fuzzer already that it becomes difficult to reason about what sort of randomness is provided in any particular use case. I wonder if it would just simpler and more readable to use fuzzFlat.

```
for (auto i = 0; i < rowType_->size(); ++i) {
   children.push_back(fuzzer.fuzzFlat(rowType_->childAt(i));
}
makeRowVector(children);
```

Or, add fuzzFlatInputRow API.

Reply:
>add fuzzFlatInputRow API

Got it, I will use this solution.

Reply:
@mbasmanova I were wondering how to convert a vector shared ptr to row vector shared ptr :)

Reply:
std::dynamic_pointer_cast